### ISSUE TITLE
Issue: Mail To Blank

### ISSUE THREAD

**Issue by LaocheXe** [e107inc/e107#2015]
Today I went to make a newsletter so what, just wanted to mass email all on the website. Went into Admin Area -> Mail and created what I wanted but wasn't allowed to select who would get the email.

![capture1](https://cloud.githubusercontent.com/assets/5818022/20112053/5e32fd60-a5b0-11e6-8ee5-768c1754fbac.PNG)

So I went and saved what I had and tried sending it, and before I hit the send button this is what the page look like:

![capture](https://cloud.githubusercontent.com/assets/5818022/20112069/769a178a-a5b0-11e6-83c5-3b2850d34070.PNG)

When trying to send it is stuck on 0% and I believe the reason why is because I couldn't select anyone to send the mail to.

**Comment by CaMer0n** [comment]
@LaocheXe Under Mail > Preferences > Email address sources- do you have something selected?

**Comment by LaocheXe** [comment]
Ah okay, selected Register Users, didn't know that was there.

**Comment by Moc** [comment]
Would it be an idea to improve this a little to make it clearer?

**Comment by CaMer0n** [comment]
@Moc Yes, I believe the label could be improved, and a check added if the value is empty. (in the actual "Compose" form)
